.babelrc

```diff
@@ -1,11 +1,22 @@
 {
   "presets": [
-    ["env", {
-      "targets": {
-        "browsers": ["> 1%", "last 2 versions", "not ie <= 8"]
+    [
+      "env",
+      {
+        "targets": {
+          "browsers": [
+            "> 1%",
+            "last 2 versions",
+            "not ie <= 8"
+          ]
+        }
       }
-    }],
-    "stage-2"
+    ],
+    "stage-2",
+    "es2015",
```

Comment:
I think the `es2015` preset is deprecated in favour of the `env` one. They do the same thing and so I think you can safely remove the `es2015` one.

Reply:
Good spot. I'll have a look at this in the morning.